doc/man1/openssl.pod

```diff
@@ -545,6 +553,36 @@ Read the password from standard input.
 
 =back
 
+=head2 Random state options
+
+Prior to OpenSSL 3.0, it was common for applications to store information
+about the state of the random-number generator in a file that was loaded
+at startup and rewritten upon exit. On modern operating systems, this is
+generally no longer necessary as OpenSSL will seed itself from the
+appropriate CPU flags, device files, and so on. These flags are still
+supported for special platforms or circumstances that might require them.
+
+It is generally an error to use the same seed file more than once and
+every use of B<-rand> should be paired with B<-writerand>.
```

Comment:
Hmm. Would it be better to enforce the presence of both (or none) options somewhere in apps.c?